model/Core/Vocabularies/RelationshipType.md

```diff
@@ -80,6 +80,7 @@ name completes the sentence:
 - republishedBy: (Security) Designates a `from` Vulnerability's details were tracked, aggregated, and/or enriched to improve context (i.e. NVD) by a `to` Agent(s)
 - serializedInArtifact: The `from` SPDXDocument can be found in a serialized form in each `to` Artifact
 - testedOn: (AI, Dataset) The `from` Element has been tested on the `to` Element
+- serviceDependsOn (Service) The `from` SoftwareService depends on each `to` SoftwareService.
```

Comment:
I'm wondering if `dependsOn` is enough to describe this relationship.

Reply:
This would simplify the spec, so I'm +1 on just using `dependsOn`.

Looking at the definition of `dependsOn`: 

"The from Element depends on each to Element during a LifecycleScopeType period."

We may need to change the definition of `LifecycleScopeType` to include service related lifecycles, like deployed.  May be a good idea to do this anyway.

Reply:
Discussed on 26 Feb 2024 - agreed to remove `serviceDependsOn` - `dependsOn` is enough.